### ISSUE TITLE
Error in running gjs

### ISSUE THREAD

**Issue by msarvi** [Alxaline/Easy-GATE-Installation#1]
Hi, 
As i run gjs command in shell i have an error core dump.  I think the problem is about gjs path for GATE

**Comment by Alxaline** [comment]
Hello,

To find the issue and replicate the error:

1) What is your distro?
`cat /proc/version`

2) Do you have any issues during the easy_gate.sh run?

3) Did you run the installation of cluster tools (HTcondor) at the end?

4) Do you have any display executing
`gjs -h`

5) If the error appear for splitting job, do you properly export the two needed environment variable:
`export GC_DOT_GATE_DIR=/somedir/`
`export GC_GATE_EXE_DIR=/usr/local/GATE/Gate-9.0-install/bin`

Thank you

**Comment by msarvi** [comment]
Dear Alxaline,
My linux version 5.4.0-58-generic and gcc version is 9.3.0 ubuntu 9.3.0-17 ubuntu1~20.04 . I had no problem in running easy_gate.sh in shell then i run HTcondor correctlly same as you but when i run gjs -h in shell i see #
![16091423175686286816043085571500](https://user-images.githubusercontent.com/48852674/103199347-e7568700-48ff-11eb-96aa-b0c7a9bc96f8.jpg)
At the end, i export both path correctly. It make me confuse.

**Comment by Alxaline** [comment]
Dear msarvi,

Thank for you reply. In order to investigate more, can you post a screenshot with the two export variable in terminal and the command line use for splitting job. Then the explicit error.

Best regards

**Comment by msarvi** [comment]
Dear Alxaline,

First of all, I just say Thank you for your help with my issue. I have a GATE code that is in a ghaebi folder which is on the desktop. I wanted to execute this code in the cluster so I had put the path of my GATE code as an "export GC_DOT_GATE_DIR ". Then change a path of "export GC_GATE_EXE_DIR" same as you. The picture below is about this part.
![Screenshot from 2020-12-29 10-47-53](https://user-images.githubusercontent.com/48852674/103304626-5cb87980-4a1e-11eb-8aff-91a7f9eab225.png)
After, I changed my path to the job splitter folder and use your instruction like below.
![Screenshot from 2020-12-29 10-55-42](https://user-images.githubusercontent.com/48852674/103304824-db151b80-4a1e-11eb-8d05-4c6d381c38b8.png)
But, I have faced this problem again. 
![Screenshot from 2020-12-29 10-56-37](https://user-images.githubusercontent.com/48852674/103305090-84f4a800-4a1f-11eb-89c3-77374b6d83c0.png)

Many thanks,
M.sarvi

**Comment by Alxaline** [comment]
Dear msarvi,

1) In top of your condor.script, do you have the variable getenv = true? :  
`gedit /usr/local/GATE/Gate-9.0/cluster_tools/jobsplitter/script/condor.script`

if not add the variable getenv = true:
```
##############################################################
#
#  Standard script for Condor in ClusterGATE 
#  Supply this script to the job splitter command line.
#  following keywords 
#   GC_EXEC
#   GC_MACRO
#  will be replaced by actual values.
#  Do not remove any of them! 
#
#  The user can edit anything before the "#GJS PART"  
#  except the line with $GC_EXEC
#
#  JDB 3.8.2005
#    
############################################################## 
getenv         = true                                                                     
Executable     = $GC_EXEC                                                   
universe       = vanilla
request_memory   = 2G
......
```
2) Is your holeGeo.mac inside your jobsplitter directory? If not put the full path.

Try again the gjs command line and tell me if it has changed.

Best regards

**Comment by msarvi** [comment]
Dear Alaxline,

1. That's true, I checked the condor. script and add `getenv = true` to the first line. 
   
![Screenshot from 2020-12-30 11-21-56](https://user-images.githubusercontent.com/48852674/103367251-e11c0280-4ad9-11eb-85e8-b804d24b10f8.png)

2.  I used the full path of my macro, then I faced to issue again.
  
![Screenshot from 2020-12-30 11-19-51](https://user-images.githubusercontent.com/48852674/103367525-92229d00-4ada-11eb-8902-acb66df3efcc.png)

I have searched Mailing List on the opengate website it was a command :
`export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/gate/Progs/Gate/gate_v6.0_p01/cluster_tools/jobsplitter/tmp/Linux-g++/gjs`.
I think the problem is the gjs path. when we use the gjs command the shell didn't run the gjs file which is located in the jobsplitter folder. would you mind, please take a look at this?. 

sincerely,
msarvi

**Comment by Alxaline** [comment]
Hello msarvi,

you forgot the `./` before the gjs command line for exec the Gate splitter.
Here a small example:

without :  `./`
![Screenshot from 2020-12-30 18-39-10](https://user-images.githubusercontent.com/43764304/103371081-dc058600-4ace-11eb-9e08-99ba65b18ccc.png)

with correct execution:  `./`
![Screenshot from 2020-12-30 18-42-13](https://user-images.githubusercontent.com/43764304/103371108-f2abdd00-4ace-11eb-8857-b92ca3325a9b.png)

Best regards

**Comment by msarvi** [comment]
Dear Alxaline,

first of all, I just can say thank you. I correct my mistake in `./gjs`. after execution again  I have (core dump) although I set environment variables as your order and add startDAQCluster to my macro. 
![Screenshot from 2020-12-30 15-10-54](https://user-images.githubusercontent.com/48852674/103379572-4e8c5b00-4afb-11eb-8726-2e1ce3535f4b.png)
![Screenshot from 2020-12-30 15-17-16](https://user-images.githubusercontent.com/48852674/103379589-5fd56780-4afb-11eb-9d4e-d1dde842326f.png)

Best regards

**Comment by Alxaline** [comment]
Dear msarvi

1) First screenshot means you don't have set the variable (maybe do not have source `GC_GATE_EXE_DIR` or `GC_DOT_GATE_DIR`. You use openmosix instead of condor and no script is used. Bad test maybe

2) Second one your right with the `./gjs`  for exec gjs  but you set a `.` for the condorscript.  So the result is an floating point exception. It's not an exec, you need the path to the script. So no `.` and just the path `-condorscript /script/condor.script`
From what i see, the command line need to be:

`./gjs  -numberofsplits 8 -clusterplatform condor -condorscript /script/condor.script  /home/messi/Desktop/ghaebi/holeGeo.mac`

Best regards

**Comment by msarvi** [comment]
Dear Alxaline,

Many thanks for your replies. I used your command line but again the `Floating point exception (core dumped)` pops. 

![Screenshot from 2021-01-02 17-45-04](https://user-images.githubusercontent.com/48852674/103459931-a27b8780-4d27-11eb-83d5-769be572754e.png)

I must note that I used `/gate/application/startDAQCluster` but I didn't used `/gate/cluster/setTimeSplitHalflife` because the 
isotope half-life is larger than acquisition time and I simulate many CTscan systems.

 
sincerely,
msarvi
